MacBook Pro 16 (2019) - i9 2.3 GHz- 32 GB - Xcode 13.3

Sherman789 opened the pull request:
## Checklist

**If you have any non-Apple hardware components - submit your results to the `Custom Hardware` table.**

* [x] I performed [all steps](https://github.com/devMEremenko/XcodeBenchmark#before-each-test) to correctly run XcodeBenchmark.
* [x] I used Xcode 12.5 or above.
* [x] I attached a screenshot with a compilation time and other fields, [example](img/contribution-example.png).
* [x] I confirm that `Time` column is still sorted.
* [x] The content in cells is centered.

<img width="812" alt="Screenshot 2022-04-04 at 20 27 31" src="https://user-images.githubusercontent.com/68182413/161610435-4f4db50b-f31b-462e-8227-7d6deeb9ebf5.png">

tylerwasick commented:
<img width="768" alt="image" src="https://user-images.githubusercontent.com/40273103/223866062-2a244026-aa2d-4e99-a5bb-831c97fe3993.png">

devMEremenko committed:
Merge pull request #432 from phatdinhvn/xcodebenchmark/macbook_m1_pro_2021_xc15

MacBook 16inch M1 Pro 16GB 512GB XC15

devMEremenko committed:
Merge branch 'master' into macbook-air-2020

devMEremenko committed:
Merge pull request #430 from ostatnicky/macbook-air-2020

Add MacBook Air 2020 M1 16GB 512GB, Xcode 15 [208 sec]

devMEremenko committed:
Merge branch 'master' into patch-1

devMEremenko committed:
Merge pull request #429 from OudomMunint/patch-1

MacBook Pro 14" 2023 M2 Pro 16GB 512GB Xcode 15, macOS 13.6, 119 seconds

devMEremenko committed:
Merge branch 'master' into macbook-pro-14-2023

devMEremenko committed:
Merge pull request #427 from ostatnicky/macbook-pro-14-2023

Add MacBook Pro 14" 2023 M2 Pro 32GB 2TB Xcode 15, 130 sec

devMEremenko committed:
Merge pull request #421 from thangldvn/master

Hackintosh: Intel i5-13500  2.5Ghz 14-core on Xcode 14.3.1

devMEremenko committed:
Merge pull request #417 from facuramirez7/facuramirez7-patch-1

Facuramirez7 Macbook Air 13" 2018

devMEremenko committed:
Merge branch 'master' into 5900x

devMEremenko committed:
Merge pull request #415 from cepheus69/5900x

Ryzentosh: AMD 5900x 4.2Ghz 12-core on Xcode 14.3.1

devMEremenko committed:
Merge branch 'master' into patch-1

devMEremenko committed:
Merge pull request #392 from lprhodes/patch-1

Add Mac Studio M1 Ultra 128GB, 2 TB SSD on Xcode 14.2

devMEremenko committed:
Merge branch 'master' into m1-macbookair

devMEremenko committed:
Merge pull request #374 from brunosxs/m1-macbookair

Added result for macbook air 2020 - 8GB RAM, 512GB SSD

devMEremenko committed:
Merge pull request #361 from julianko13/master

Add line to read.me file including MacBook Pro 16" 2021 running xcode 14 version

devMEremenko committed:
Merge pull request #345 from kashifhisam/master

Add VMWare Player 2.77ghz 16gb 8c / Zotac Magnus One ECM73070C i7-10700 (8c 2.9ghz) - 32GB ram - 1TB ssd

devMEremenko committed:
Update ReadMe.md

devMEremenko committed:
Merge pull request #321 from andrei-bucurei/mac-mini-2012-i5-open-core-legacy-patch

Add Mac Mini Late 2012, dual-core Intel i5 @ 2.5GHz, 16GB RAM, SSD 256GB, Mac OS 12.3, Xcode 13.3, Open Core Legacy Patcher

devMEremenko committed:
Update ReadMe.md

devMEremenko committed:
Merge branch 'master' into master

devMEremenko committed:
Merge pull request #366 from DanielDanaee/master

MacBook Air 13" 2020 M1 8GB Xcode 14.0.1

devMEremenko committed:
Update ReadMe.md

devMEremenko committed:
Update ReadMe.md

devMEremenko committed:
Update ReadMe.md

devMEremenko committed:
Update ReadMe.md

markguleno committed:
Update ReadMe.md

Fix typo

devMEremenko committed:
Merge pull request #451 from markguleno/6-core

bilyksergii committed:
Fixed example image

bilyksergii committed:
Merge branch 'master' into master

devMEremenko committed:
Merge pull request #446 from bilyksergii/master

Macbook Pro 16" 2021 M1Pro 32GB 1TB

devMEremenko committed:
Update ReadMe.md

devMEremenko committed:
Update ReadMe.md

devMEremenko committed:
Update ReadMe.md

devMEremenko committed:
Update ReadMe.md

Based on https://github.com/devMEremenko/XcodeBenchmark/issues/453

devMEremenko committed:
Update ReadMe.md

Based on https://github.com/devMEremenko/XcodeBenchmark/issues/444

valerio8 committed:
Update ReadMe.md

pharshdev committed:
Update ReadMe.md

16" MBP M3 Pro 12 Core (6P, 6E) 36 GB in 110s

iamszabo committed:
Update ReadMe.md

x3mall1986 committed:
Update ReadMe.md

basoppenheim committed:
Update ReadMe.md

basoppenheim committed:
Update ReadMe.md

nullskill committed:
Mac Studio 2022 M1 Max 10 Core 32GB 512GB | 134s

luisppb16 committed:
Update ReadMe.md

kdurant35 committed:
Update ReadMe.md

Alex-pvl committed:
Add M2 Pro 16 inch

gubnota committed:
M1 16/1TB Xcode 15.2

fweez committed:
Add MBP 14" 2021, Xcode 15.2

IkerPerea committed:
Update ReadMe.md

IkerPerea committed:
Update ReadMe.md

IkerPerea committed:
Update ReadMe.md

badepaz committed:
Update ReadMe.md to add new result

Added a new result for my new MacBook Pro 16” 16-core set to high power mode.

badepaz committed:
Added New Result for 2023 MacBook Pro M3 Max - Fixed Typo

Fixed a typo. Previously the result said 2022 when it was supposed to be 2023.

JohnBehnke committed:
Add Mac Studio

levochkaa committed:
added column

OnurVar committed:
Updated with MacBook Pro M3 Pro 11-Core

ostatnicky committed:
Update ReadMe.md

ostatnicky committed:
Update ReadMe.md

ostatnicky committed:
Update ReadMe.md

devMEremenko committed:
Merge pull request #503 from ostatnicky/macbook-air-2024

MacBook Air 13" 2024 M3 8C 24GB 2T | 140 sec

devMEremenko committed:
Merge pull request #502 from OnurVar/master

Macbook Pro 14" 2023 M3 Pro | Xcode 15 - macOS 14 | 108s

devMEremenko committed:
Merge branch 'master' into m2max

devMEremenko committed:
Merge pull request #501 from levochkaa/m2max

Macbook Pro 16" 2023 M2 Max | Xcode 15 - macOS 14 | 114s

devMEremenko committed:
Merge pull request #498 from JohnBehnke/macstudio

Mac Studio 2022, M1 Ultra 20c, 128GB, Sonoma 14.4, Xcode 15.3 [73.060-sec]

devMEremenko committed:
Merge pull request #496 from badepaz/master

Update ReadMe.md to add new result for MacBook

devMEremenko committed:
Merge pull request #494 from IkerPerea/master

Xcode Benchmark - Hackintosh - i5 10400 2.9Ghz 6-Core

devMEremenko committed:
Merge branch 'master' into fweez-mbp14-2021

devMEremenko committed:
Merge pull request #491 from fweez/fweez-mbp14-2021

MacBook Pro 14" 2021 M1 Pro 8 core 32GB 1TB | Xcode 15.2, macOS 14.1 | 170s

devMEremenko committed:
Merge branch 'master' into patch-2

devMEremenko committed:
Merge pull request #489 from gubnota/patch-2

M1 16/1TB Xcode 15.2

devMEremenko committed:
Merge branch 'master' into master

devMEremenko committed:
Merge pull request #488 from Alex-pvl/master

Macbook Pro 16" 2023 M2 Pro 12 Cores 16GB 512GB | 111s

devMEremenko committed:
Merge branch 'master' into m2max

devMEremenko committed:
Merge pull request #477 from kdurant35/m2max

Macbook Pro 16" 2023 M2 Max | Xcode 15 - macOS 14 | 110s

devMEremenko committed:
Merge branch 'master' into master

devMEremenko committed:
Merge pull request #474 from luisppb16/master

Macbook Pro 16" 2023 M3 Pro 12-Core (6p, 6e) 18GB 512 | 104s

devMEremenko committed:
Merge branch 'master' into mac-studio-2022

devMEremenko committed:
Merge pull request #469 from nullskill/mac-studio-2022

Mac Studio 2022 M1 Max 10 Core 32GB 512GB | 134s

nnsnodnb committed:
Add Mac Studio 2023 M2 Ultra RAM:128 SSD:2TB Xcode:15.4 macOS:15.4

devMEremenko committed:
Fixed an issue with BoringSSL on Xcode 16

devMEremenko committed:
Added Xcode 16 section to the ReadMe

KAnggara75 committed:
add: XCode 16 MBP 2020 (M1 8c) 16GB

Model Name: MacBook Pro
Model Identifier: MacBookPro17,1
Total Number of Cores: 8 (4 performance and 4 efficiency)
Memory: 16 GB

devMEremenko committed:
Merge pull request #515 from KAnggara75/master

add: XCode 16 MBP 2020 (M1 8c) 16GB

iuriipleskach committed:
add: XCode 16 MBP 2021 (M1 Max 10c) 32GB

devMEremenko committed:
Merge pull request #516 from iuriipleskach/master

Macbook Pro 16" 2021 M1 Max 10 Cores 32GB 1TB | 152s | Xcode 16.1 | macOS 15.1

spencekl99 committed:
Update ReadMe.md

iuriipleskach committed:
add: XCode 16 MBP 2023 (M3 Max 14c) 36GB

devMEremenko committed:
Merge pull request #518 from iuriipleskach/master

Macbook Pro 16" 2023 M3 Max 14 Cores 36GB 1TB | 110s | Xcode 16.1 | macOS 14.7.1

devMEremenko committed:
Merge branch 'master' into master

devMEremenko committed:
Merge pull request #517 from spencekl99/master

Mac Pro 2023 24 Cores 192GB 8TB | 87s | Xcode 16.1 | macOS 15.1

madcato committed:
add: XCode 16.1 Mac Mini 2023 (M2 Pro 10c) 16GB

madcato committed:
add: Xcode 16.1 MacBook Air 2024 (M3 8c) 8GB

devMEremenko committed:
Merge pull request #519 from madcato/mac-mini-M2-Pro-16G-xcode-16.1

XCode 16.1 Mac Mini 2023 (M2 Pro 10c) 16GB

devMEremenko committed:
Merge pull request #520 from madcato/macbook-air-M3-8GB-xcode16.1

Xcode 16.1 MacBook Air 2024 (M3 8c) 8GB

devMEremenko committed:
Merge branch 'master' into master

devMEremenko committed:
Merge pull request #466 from BasO/master

MacBook Air 13" 2022 M2 (10-core GPU) 8GB 512GB - 158

bensinther committed:
Update ReadMe.md

add test results for M3 Max 16 Core 128GB

devMEremenko committed:
Update ReadMe.md

devMEremenko committed:
Merge pull request #527 from devMEremenko/devMEremenko-patch-1

Mac mini 2024, M4 Pro

devMEremenko committed:
Merge pull request #456 from valerio8/master

Mac Mini 2018, i3 3.6Ghz, 16GB, 256GB

devMEremenko committed:
Merge branch 'master' into 16-MBP-M3Pro-12C-36GB-1

devMEremenko committed:
Merge pull request #459 from pharshdev/16-MBP-M3Pro-12C-36GB-1

16" MBP M3 Pro 12 Core (6P, 6E) 36 GB | 110s

devMEremenko committed:
Merge branch 'master' into master

devMEremenko committed:
Merge pull request #525 from bensinther/master

Macbook Pro 16" 2023 M3 Max 16 Cores 128GB 1TB | 90s

witekbobrowski committed:
Add M1 Ultra Mac Studio (2022) results

devMEremenko committed:
Merge pull request #528 from witekbobrowski/master

Add M1 Ultra Mac Studio (2022) results

devMEremenko committed:
Update ReadMe.md

basoppenheim committed:
Update ReadMe.md

Add MacBook Pro 14" 2023 (M3 Pro 11-core) result for Xcode 16.

mcichecki committed:
update readme with MBP 14 M4 Pro

lprhodes committed:
Added Mac Mini M4 Pro 14 Core

witekbobrowski committed:
Add M2 MacBook Air (2022) results

devMEremenko committed:
Merge pull request #529 from BasO/master

MacBook Pro 14" 2023 M3 Pro 11 Cores 18GB 512GB | 140s | Xcode 16.1 | macOS 15.1

devMEremenko committed:
Merge pull request #530 from mcichecki/master

Add result for MacBook Pro 14" with base M4 Pro

Commit:
Add Mac Mini 10-core 16/256GB

Tomyail committed:
Add result for MacBook Pro 14" with  M4 Pro (14 core 48GB)

Commit:
Add mac-mini M4 Pro (14-cores) 64GB

alexziskind1 committed:
added M4 MacBook Pro 14" 2024 results

alexziskind1 committed:
added M4 Max MacBook Pro 16" 2024 results

alexziskind1 committed:
added M4 Pro MacBook Pro 14" 2024 results

alexziskind1 committed:
Added M2 Max MacBook Pro 16" 2022 results

devMEremenko committed:
Merge branch 'master' into mac-mini-m4-pro-14

devMEremenko committed:
Merge pull request #531 from lprhodes/mac-mini-m4-pro-14

Added Mac Mini M4 Pro 14 Core

devMEremenko committed:
Merge pull request #533 from witekbobrowski/master

Add M2 MacBook Air (2022) results

devMEremenko committed:
Merge pull request #534 from kondratk/mac-mini-m4

Add Mac Mini 2024 M4 10-core 16GB/256GB | 141s

devMEremenko committed:
Merge branch 'master' into master

devMEremenko committed:
Merge pull request #536 from Tomyail/master

 MacBook Pro 14" with  M4 Pro (14 core 48GB 512GB) [97.792 sec]

devMEremenko committed:
Merge branch 'master' into M4-MacBook-Pro-14-2024-results

devMEremenko committed:
Merge pull request #545 from alexziskind1/M4-MacBook-Pro-14-2024-results

added M4 MacBook Pro 14" 2024 results

devMEremenko committed:
Merge pull request #546 from alexziskind1/M4Max-MacBook-Pro-16-2024-results

added M4 Max MacBook Pro 16" 2024 results

devMEremenko committed:
Merge branch 'master' into M4-Pro-MacBook-Pro-14-2024-Results

devMEremenko committed:
Merge pull request #547 from alexziskind1/M4-Pro-MacBook-Pro-14-2024-Results

added M4 Pro MacBook Pro 14" 2024 results

devMEremenko committed:
Merge pull request #548 from alexziskind1/M2Max-MacBook-Pro-16-2022-results

Added M2 Max MacBook Pro 16" 2022 results

devMEremenko committed:
Merge branch 'master' into mac-mini-M4-Pro-64G-Xcode-16.1

devMEremenko committed:
Merge pull request #540 from madcato/mac-mini-M4-Pro-64G-Xcode-16.1

Mac-mini M4 Pro (14-cores) 64GB

ShenaRingo326 committed:
Run benchmark

stepanic committed:
Mac-mini-M4-Pro-12CPU-16GPU-24GB-512-Xcode-16.1-MacOS-15.1

stepanic committed:
Mac-mini-M4-Pro-12CPU-16GPU-24GB-512-Xcode-16.1-MacOS-15.1-103s

stevenpearson committed:
Update with Mac Mini 2018 i7 result

bbaic committed:
add imac data to table

devMEremenko committed:
Merge pull request #553 from stepanic/master

Mac-mini-M4-Pro-12CPU-16GPU-24GB-512-Xcode-16.1-MacOS-15.1-103s

anvein committed:
add macbook pro m3 pro 12 core

roanutil committed:
MBP16 M4 Max 16 core 64GB 2TB

mbp16-2024-m4-max-16CPU

roanutil committed:
MBP16 M2 Max 12 core 64GB 2TB

mbp16-2023-m2-max-12CPU

razzededge committed:
Results for Macbook Pro M4 Max 16 Inch 2024 16 Core 64 GB 1 TB

Commit:
added mba-m3-16

devMEremenko committed:
Added the github folder

devMEremenko committed:
Update ReadMe.md

devMEremenko committed:
Update ReadMe.md

devMEremenko committed:
Merge pull request #567 from razzededge/master

Results for Macbook Pro M4 Max 16 Inch 2024 16 Core 64 GB 1 TB

devMEremenko committed:
Update ReadMe.md

devMEremenko committed:
Merge pull request #568 from braginets/master

Added MacBook Air 13" 2024 M3, 16GB, MacOS 14.6, Xcode 16.2 [184.022 sec]

devMEremenko committed:
Update ReadMe.md

devMEremenko committed:
Merge pull request #566 from roanutil/mbp16-2023-m2-max-12CPU

MBP16 M2 Max 12 core 64GB 2TB

devMEremenko committed:
Merge branch 'master' into mbp16-2024-m4-max-16CPU

devMEremenko committed:
Merge pull request #565 from roanutil/mbp16-2024-m4-max-16CPU

MBP16 M4 Max 16 core 64GB 2TB

devMEremenko committed:
Merge pull request #557 from bbaic/baic-imac

iMac 19,1 8-core i9, 24 GB, 512 SSD

devMEremenko committed:
Merge pull request #556 from steven-pearson/2018MacMiniI7

Update with Mac Mini 2018 i7 result

devMEremenko committed:
Merge branch 'master' into master

devMEremenko committed:
Merge pull request #561 from anvein/master

MacBook Pro 16" M3 pro, 12 core, 36gb, 512gb ssd, XCode 16.1, macOS 15.1.1 | 134s

devMEremenko committed:
Merge branch 'master' into master

devMEremenko committed:
Merge pull request #552 from ShenaRingo326/master

Mac Studio 2023 M2 Max 12 Cores 32GB 512GB | 130s

devMEremenko committed:
Merge branch 'master' into master

devMEremenko committed:
Merge pull request #465 from x3mall1986/master

Macbook Pro 16" 2021 M1 Pro 10 Cores 16GB 512GB | 133s

seannam committed:
Add MBP 14 2021 M1 Pro

devMEremenko committed:
Update ReadMe.md

devMEremenko committed:
Merge pull request #570 from seannam/master

Macbook Pro 13" 2021 M1 Pro 10 Cores 32GB 1TB | 184s

huntleask committed:
Update ReadMe.md

aravhawk committed:
Add M3 Pro 12-core MacBook Pro 14"

kriuchkov-guava committed:
Add new result to ReadMe file

taf-fs committed:
Add MacBook Air 13" 2020

frankrei committed:
MacStudio 2025 28 core

joehinkle11 committed:
Mac Studio 2025

Commit:
Mac Studio 2025 M3 Ultra, Cores: 32,Memory: 512 GB, Sequoia 15.3.2, Xcode 16.2 [66.980 sec]

Commit:
Add Ryzen 9950X to ReadMe.md

devMEremenko committed:
Merge pull request #582 from joehinkle11/master

Mac Studio 2025

devMEremenko committed:
Merge branch 'master' into master

devMEremenko committed:
Merge pull request #584 from yidaizhenlong/master

Mac Studio 2025 M3 Ultra, Cores: 32,Memory: 512 GB, Sequoia 15.3.2, X…

devMEremenko committed:
Merge branch 'master' into frankrei-patch-1

devMEremenko committed:
Merge pull request #580 from frankrei/frankrei-patch-1

MacStudio 2025 28 core

devMEremenko committed:
Merge pull request #577 from bllizard22/master

M4 Pro 14-core MacBook Pro 16"

devMEremenko committed:
Merge pull request #587 from azeeka/master

Add Ryzen 9950X to ReadMe.md

devMEremenko committed:
Merge pull request #575 from aravhawk/master

Add M3 Pro 12-core MacBook Pro 14"

devMEremenko committed:
Merge pull request #579 from taf-fs/master

Add MacBook Air 13" 2020, M1

devMEremenko committed:
Merge pull request #572 from huntleask/patch-1

Mac Studio (12c M2 Max, 64GB, 1TB)

devMEremenko committed:
Update ReadMe.md

botjaeger committed:
chore: update README

mhahnFr committed:
Added result for iMac19,2 21.5" 2019

devMEremenko committed:
Updated pods

devMEremenko committed:
Updated gitIgnore

devMEremenko committed:
Added new section for Xcode 16 results

devMEremenko committed:
Added M4 Max 14-core

devMEremenko committed:
Merge pull request #593 from mhahnFr/master

Added result for iMac19,2 21.5" 2019

devMEremenko committed:
Update ReadMe.md

klemti committed:
Update ReadMe.md

devMEremenko committed:
Merge pull request #601 from klemti/master

aryanvx committed:
Update ReadMe.md

added chart for Xcode 26; test conducted on m4 air running tahoe

devMEremenko committed:
Merge pull request #609 from aryanvx/master

Update ReadMe.md

devMEremenko committed:
Merge pull request #512 from nnsnodnb/master

Mac Studio 2023 M2 Ultra 24 Cores 128GB 2TB | 65s

devMEremenko committed:
Merge pull request #588 from botjaeger/patch-1

Macbook Pro M3 Max 14-inch result

devMEremenko committed:
Merge branch 'master' into master

devMEremenko committed:
Merge pull request #462 from iamszabo/master

Macbook Pro 13" 2021 M1 Pro 8 Cores 32GB 512GB | 175s

devMEremenko committed:
Update ReadMe for Xcode 16 support

Removed previous Xcode 16 results section and updated device listing.

devMEremenko committed:
Update device specifications for Xcode 26

devMEremenko committed:
Add Macbook Pro 16" 2024 with M2 Pro specs

aryanvx committed:
Update ReadMe.md

aryanvx committed:
Update ReadMe.md

devMEremenko committed:
Merge pull request #611 from aryanvx/master

Xcode 26.1 - Mac Mini M2 Pro 16GB / 512GB

jordanhom committed:
docs: Update ReadMe with Xcode 26.1 MacBook Air M1 results.

Commit:
Add Mac Studio 2023 with M2 Ultra specifications to ReadMe.md

jonauz committed:
MacBook Pro 14" 2023, M2 Pro 10c, RAM 16GB, SSD 512GB, Xcode 26.3, Tahoe 26.3

jonauz committed:
MacBook Pro 14" 2024, M4 Pro 12c, RAM 24GB, SSD 512GB, Xcode 26.3, Tahoe 26.3

jonauz committed:
Fix last update of ReadMe.md

Incorrect CPU description. Should be M4 instead of M2.

devMEremenko committed:
Update Xcode version requirement in ReadMe

devMEremenko committed:
Merge pull request #625 from codeyogi911/master

Add Mac Studio 2023 with M2 Ultra specifications to ReadMe.md

devMEremenko committed:
Added M1 Ultra

Added Mac Studio 2022 entry with M1 Ultra specs.

Based on https://github.com/devMEremenko/XcodeBenchmark/pull/614/changes#diff-0b59e9c9f474883e24cb1319da596e81aa164472c07d90db0df74948792cf6cbR17

devMEremenko committed:
Merge pull request #618 from jordanhom/wip-m1ba

Xcode 26.1 - MacBook Air (2020) - M1 - 8GB / 256GB

devMEremenko committed:
Merge branch 'master' into master

devMEremenko committed:
Merge pull request #633 from jonauz/master

Add MacBook Pro 14" 2023 M2 16GB and MacBook Pro 14" 2024 M4 24GB

devMEremenko committed:
ReadMe update

Removed the Mac mini 2023 entry from the specifications table because it most likely contains stale results (due to the recent XcodeBenchmark update). Waiting for new submissions.

Hans92 committed:
ReadMe update

devMEremenko committed:
Merge pull request #638 from Hans92/master

MacBook Pro 16" 2026 M5 Max 18-core - Xcode 26.3

Hans92 committed:
ReadMe update

devMEremenko committed:
Merge pull request #639 from Hans92/master

MacBook Pro 16" 2021 M1 Max 10-core - Xcode 26.3

devMEremenko committed:
Update ReadMe.md

Removed M2 Pro with 344 sec result because it looks like the test was conducted with a previous version of XcodeBenchmark, or under unusual conditions. Waiting for more results.

michallaskowski committed:
Add MacBook Pro M4 Max 16c

devMEremenko committed:
Merge pull request #640 from michallaskowski/master

Add results for MacBook Pro M4 Max 16c, Xcode 26.3 (89 seconds)

kflip committed:
macbook air 13 m5 2026 10cpu 10gpu 32gb 512gb

torgr1m committed:
Update ReadMe.md

devMEremenko committed:
Merge pull request #642 from kflip/master

Macbook Air 13 M5 2026 10cpu 10gpu 32gb 512gb (173 sec)

devMEremenko committed:
Merge pull request #643 from torgr1m/master

Update ReadMe.md

harryworld committed:
Update ReadMe.md

harryworld committed:
Update ReadMe.md

harryworld committed:
Update ReadMe.md

q1angch0u committed:
add M1 Pro 8c 16g data

devMEremenko committed:
Merge pull request #644 from harryworld/master

MacBook Pro 16" 2026 M5 Pro 18-core - Xcode 26.3

devMEremenko committed:
Merge pull request #645 from ryanbuu/m1-pro-8c-16g

add M1 Pro 8c 16g data

aliosmanozturk1 committed:
Add Macbook Pro 14" 2021 M1 Pro

andimasbagaswara committed:
Add Macbook Pro 14" 2025 specifications

andimasbagaswara committed:
Add Macbook Pro 14" 2025 entry to ReadMe

devMEremenko committed:
Merge pull request #648 from andimasbagaswara/master

MacBook Pro 14" 2025 M5 10-core - Xcode 26.3

devMEremenko committed:
Remove duplicate MacBook Pro 14" 2021 entry

Removed duplicate entry for MacBook Pro 14" 2021.

devMEremenko committed:
Merge pull request #646 from aliosmanozturk1/master

Add MacBook Pro 14" 2021 (M1 Pro 10-core) benchmark result (Xcode 26.3)

ishaan1995 committed:
Added MacBook Neo benchmark

sunimp committed:
Fix Xcode 26.4 benchmark build

sunimp committed:
Add Mac mini M4 Pro Xcode 26.4 result

devMEremenko committed:
Merge pull request #650 from sunimp/sun-fix-xcode-26_4-afnetworking

Fix Xcode 26.4 benchmark build

sarpsolakoglu committed:
add m5 pro 15 core measurement

devMEremenko committed:
Merge pull request #649 from ishaan1995/patch-1

MacBook Neo 13" - A18 Pro - 8GB benchmark

devMEremenko committed:
Merge pull request #654 from sarpsolakoglu/m5pro-15core

Add Macbook Pro 14" M5 Pro 15-core 24GB 1TB Xcode 26.4 Measurement

devMEremenko committed:
Merge pull request #651 from sunimp/sun-result-pr-mac-mini-m4-pro-xcode-26_4

Add Mac Mini 2024 M4 Pro Xcode 26.4 result
